IMS.BL/Inventory.cs

```diff
@@ -2,51 +2,42 @@
 using System.Linq;
 using AutoMapper;
 using System;
+using IMS.BL.DataService;
+using IMS.DL.SqlDatabaseConnection;
 
 namespace IMS.BL
 {
     public class Inventory
```

Comment:
so this is a repository ? names it InventoryRepository also you need to apply strategy pattern for this project, re structure it as a web app